react-hash-routing/client/components/header.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function Header(props) {
+  return (
+    <header className="mb-5">
+      <nav className="navbar navbar-dark bg-dark shadow-sm">
+        <div className="container">
+          <div className="col px-0">
+            {/* this anchor should go back to the catalog at '#' */}
+            <a href="http://localhost:3000/#" className="navbar-brand">
```

Comment:
Here we don't want to include `http://localhost:3000` should just be just `href="#"`